linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -2112,6 +2121,13 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
               method,
               transformTypedArgs(method.name, args))
 
+        case ApplyDynamicImport(flags, className, method, args) =>
+          val call = withDynamicGlobalVar("s", (className, method.name)) { v =>
+            js.Apply(v, transformTypedArgs(method.name, args))
```

Comment:
This translation is a bit too naive. It only works if all the `args` are *pure* expressions (side-effect-free is not even enough). If they're not, we need to make sure to always evaluate the `args` *before* performing the `ImportCall`. The semantics of `ApplyDynamicImport(..., method, args)` should be:

1. Evaluate all the `args` as `argsValues`
2. Asynchronously load the module and execute `method(...argsValues)`